File: Python Codes/Odd_Even A22.py
#Python Program to demonstrate odd_even number
num = input("Enter an integer: ")

#using exception handling
try:
    number = int(num)
    if number % 2 == 0:
        print(f"{number} is even.") #f string formatter
    else:
        print(f"{number} is odd.") #f string formatter
except ValueError:
    print("Enter valid data. The input is not a valid integer.")

File: Odd_Even A22.c
// C Program to demonstarte odd_even Number
// By Amol Rakh

#include <stdio.h>

int main()
{
    int num;
    printf("Enter a number : ");
    scanf( "%d",&num);
    if ( num % 2 == 0 ) // Comparison opreater for odd-even number
    {
        printf("%d is Even number", num);
    }
    else
    {
        printf("%d is Odd number", num);
    }
    return 0;
    
}